### DISCUSSION TITLE
Icon Submissions for Win98SE

### DISCUSSION THREAD

**Discussion by malekmasoud** [nestoris/Win98SE#3] [Show and tell]
Here I arrive with some icon submissions that I came up with, including...

Earth Globe (60, 72 and 96; the globe was extracted from xml-file)
![Globe 60](https://user-images.githubusercontent.com/25880200/158044347-31cb8d5e-a4b2-4b15-805d-eed69133dc5c.png) ![Globe 72](https://user-images.githubusercontent.com/25880200/158044348-29bb7b17-7b4e-43aa-9ca5-b11c5428c37c.png) ![Globe 96](https://user-images.githubusercontent.com/25880200/158044351-c74762ee-6aee-451d-803a-2334d5bfe18e.png)

Earth Globe with Mail (16, 32, 48, 64, 72 and 96)
![Mail Globe 16](https://user-images.githubusercontent.com/25880200/158044389-a41057e8-fc4c-4b25-b4f9-298001136daa.png) ![Mail Globe 32](https://user-images.githubusercontent.com/25880200/158044390-4086223f-26c2-4413-b898-797f15536570.png) ![Mail Globe 48](https://user-images.githubusercontent.com/25880200/158044393-5ff6a3a8-a293-4a3b-86c8-db1aaecf0e13.png) ![Mail Globe 64](https://user-images.githubusercontent.com/25880200/158044394-649c512b-8e3d-42af-9323-fda19564f6c5.png) ![Mail Globe 72](https://user-images.githubusercontent.com/25880200/158044395-88d24586-ec80-4b5e-8f3e-3d41e12353ee.png) ![Mail Globe 96](https://user-images.githubusercontent.com/25880200/158044396-cbf68cfb-9772-41ab-b7f8-0ccc422e3c61.png)

Outlook Express (64, 72 and 96)
![Outlook Express 64](https://user-images.githubusercontent.com/25880200/158046180-f5cc1ec8-4ad8-4d6d-97a9-6de680467267.png) ![Outlook Express 72](https://user-images.githubusercontent.com/25880200/158046181-116460a4-3203-49f9-b58d-67155a9d3064.png) ![Outlook Express 96](https://user-images.githubusercontent.com/25880200/158046178-4e475955-27a4-415e-a9d3-7507084b7e74.png)

**Comment by nestoris** [comment]
Cool! Thank you! I try to use the limited palette of 256 colors extracted from Windows 98 icons and diffuse (noise) blending (as in Adobe Photoshop(R) 5.0-7.0) instead of true-color gradients. So I understand your desire and I will try to realize it.
P.S. I already have a Globe.ico with various formats in it :)
I wrote some front-ends for my convenience, to extract icon formats directly into the theme, to compile 'ico' format icon from png files, for quickly making hundreds of symlinks (when same image is used for various icon names). You can join the project (fork and suggest commits) if you follow some style rules. I can explain by [e-mail](mailto:voltasar@gmail.com) how to achieve this with little effort with the help of some Windows programs and my Linux scripts.

**Reply by malekmasoud** [reply]
Yep! I would sure love to see that e-mail of yours! (my email address is malekmasoud3@gmail.com)

Oh, and for those wondering, I actually use GIMP to make these icons.

**Reply by nestoris** [reply]
Now I try to write [reference for rescaling and creating new icons in 98's style](https://github.com/nestoris/Win98SE/tree/main/reference).
If your native language is not English and it's not present in theme descriptions in index.theme file, you may add your translation there and make pull-request or write it here.

**Reply by nestoris** [reply]
If you don't mind, I'll use some your images to show to all future editors what is correct and what's incorrect. :)

**Comment by nestoris** [comment]
Also I need help from a programmers: I need an 'index-color' filter plugin for Krita for making a filter-layer that will convert the editing multi-layered image (or a part of image chosen by a mask) to an indexed (with custom pallette) color view with pattern or diffused mixing. If you know somebody and he want's to help -- it will be very good! :)

**Comment by malekmasoud** [comment]
"You've Got Mail", as quoted by AOL

Here I am, ready to mail in the improved versions of my Outlook Express that have been indexed to fit the 256-color Windows 98 palette. Guess I need to start checking my e-mailbox more often, huh?
![Outlook Express 64](https://user-images.githubusercontent.com/25880200/158826299-4b0b6a46-5bda-455f-a42c-36e1a4fdf9c2.png) ![Outlook Express 72](https://user-images.githubusercontent.com/25880200/158826300-0e8ae4b5-9fa0-4471-a218-24a92a3935be.png) ![Outlook Express 96](https://user-images.githubusercontent.com/25880200/158826296-62bf4f45-2485-40e2-a5fb-582d5721ddad.png)

**Comment by malekmasoud** [comment]
Here's the old Outlook Express icon that has the Internet Explorer logo in sizes ranging from 16px all the way 256px.
![Outlook Express 4 16](https://user-images.githubusercontent.com/25880200/159976614-a69362b3-acd9-4341-bd07-8e6d78bd0406.png) ![Outlook Express 4 32](https://user-images.githubusercontent.com/25880200/159976776-98c19733-ecfc-4649-8602-b1a8d013d61d.png) ![Outlook Express 4 48](https://user-images.githubusercontent.com/25880200/159976798-8bce30d3-3bbf-4a6c-a233-8ae3f756d261.png) ![Outlook Express 4 64](https://user-images.githubusercontent.com/25880200/159976873-a34a5f8b-a4b4-44fb-bbfb-3720797a8559.png) ![Outlook Express 4 72](https://user-images.githubusercontent.com/25880200/159976905-bb42a920-25ac-4e79-a766-0146c78aa507.png) ![Outlook Express 4 96](https://user-images.githubusercontent.com/25880200/159976933-68813ddd-de9f-4e49-a756-94bc0c54ae71.png) ![Outlook Express 4 128](https://user-images.githubusercontent.com/25880200/159976976-c4124cda-d949-4c71-b15a-21db6482a50a.png)  ![Outlook Express 4 256](https://user-images.githubusercontent.com/25880200/159977012-501e9adc-f9b5-459c-aee8-9bc1f474093e.png)

**Reply by nestoris** [reply]
Cool! Too many alternatives for e-mail icon. Maybe somebody prefer one and don't prefer others... Maybe it'll good to have something like  Python script with GUI installation as in Chicago95 project for choosing preferred icon. Anyway it is rare when the larger then 96x96 px. sizes are used for application icon. Large sizes are useful for Mime types, places, and some devices in computer:/// directory. If you want this icon to be a bit corrected and applied to the Theme please send me it as multi-layered Gimp files by e-mail. I use multi-layer editor for Windows icons.

**Reply by nestoris** [reply]
the internet-mail icon is with this image. (shadow maybe will be soon)
![16x16](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/16/internet-mail.png) ![22x22](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/22/internet-mail.png) ![24x24](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/24/internet-mail.png) ![32x32](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/32/internet-mail.png) ![48x48](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/48/internet-mail.png) ![64x64](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/64/internet-mail.png) ![96x96](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/96/internet-mail.png) ![128x128](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/128/internet-mail.png) ![160x160](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/160/internet-mail.png) ![192x192](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/192/internet-mail.png) ![256x256](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/256/internet-mail.png)

**Comment by nestoris** [comment]
Here are the icons of OE for Mozilla Thunderbird:
[64x64](https://github.com/nestoris/Win98SE/raw/main/SE98/apps/64/thunderbird.png)
[72x72](https://github.com/nestoris/Win98SE/raw/main/SE98/apps/72/thunderbird.png)
[96x96](https://github.com/nestoris/Win98SE/raw/main/SE98/apps/96/thunderbird.png)

**Comment by malekmasoud** [comment]
Here is the Globe icon in a 128x128px size
![Globe 128](https://user-images.githubusercontent.com/25880200/213887531-b9924c9b-ab3c-4087-9062-e298b04f819d.png)

**Comment by malekmasoud** [comment]
Windows Update icons ahead!
![Windows Update 64](https://user-images.githubusercontent.com/25880200/213934650-78bc024e-054d-42bb-883b-d93910fe1dd1.png) ![Windows Update 72](https://user-images.githubusercontent.com/25880200/213934651-536ac1c5-6fd9-4d42-a353-398adbb82a37.png) ![Windows Update 96](https://user-images.githubusercontent.com/25880200/213934653-8aabb808-f1c8-48e0-ac8b-69301b6c8f1e.png) ![Windows Update 128](https://user-images.githubusercontent.com/25880200/213934655-0e156e75-8180-4982-a9da-eb04c8732d68.png) ![Windows Update 160](https://user-images.githubusercontent.com/25880200/213934657-3cf1f954-11e1-4589-9243-baa1ade48862.png) ![Windows Update 192](https://user-images.githubusercontent.com/25880200/213934649-90ed44c6-e8d5-4568-9a4b-69b1552f0f08.png)

**Comment by malekmasoud** [comment]
Another classic 3.x icon has been 98-ified and this time, it's Paintbrush!
![Paintbrush 16](https://user-images.githubusercontent.com/25880200/213937452-295a1865-47ed-42e2-b098-a19b90d762a8.png) ![Paintbrush 32](https://user-images.githubusercontent.com/25880200/213937454-dcbc52a0-a03b-4ec4-b7d1-fd41af78ea2b.png) ![Paintbrush 48](https://user-images.githubusercontent.com/25880200/213937451-42d5a416-5810-4ae1-8d1f-54db647beb87.png)

**Comment by nestoris** [comment]
done. except Paintbrush. Paintbrush -- a bit later

**Comment by malekmasoud** [comment]
Here I come with two networking icons that have sizes ranging from 16x16 to 96x96

**Network Error/Limited Connection (No Internet Access)**
![Network Error 16](https://user-images.githubusercontent.com/25880200/220017867-978ec0cd-ef66-4aad-869d-c45952ac72fe.png) ![Network Error 22](https://user-images.githubusercontent.com/25880200/220017871-736aa106-5dd3-4f39-b40b-7a08ab1bd6b6.png) ![Network Error 24](https://user-images.githubusercontent.com/25880200/220017872-981755ce-3180-4b8a-b640-0a1edeb95277.png) ![Network Error 32](https://user-images.githubusercontent.com/25880200/220017873-fe6ec905-d3a4-4047-8e52-2f52f7ca75ab.png) ![Network Error 48](https://user-images.githubusercontent.com/25880200/220017880-00b32568-77db-4c5e-b9b3-aec121037725.png) ![Network Error 64](https://user-images.githubusercontent.com/25880200/220017883-060a8809-95e1-4dc9-9d5e-149052d4005f.png) ![Network Error 72](https://user-images.githubusercontent.com/25880200/220017887-d21d5780-9aa4-4e06-9236-86a7d2b72ce0.png) ![Network Error 96](https://user-images.githubusercontent.com/25880200/220017889-724863ef-9a34-4c07-8b3e-8b09e3e06059.png)

**Network Offline (Disconnected)/No Connection**
![Network Disconnected 16](https://user-images.githubusercontent.com/25880200/220018148-7ac484bb-302e-4528-82a4-5c3aeb9890d3.png) ![Network Disconnected 22](https://user-images.githubusercontent.com/25880200/220018151-77b1ce72-c041-4395-b473-c5331ada4d02.png) ![Network Disconnected 24](https://user-images.githubusercontent.com/25880200/220018154-61320ddb-05d4-4f4a-a348-222bda2b118a.png) ![Network Disconnected 32](https://user-images.githubusercontent.com/25880200/220018156-28e810d7-c75c-483b-828e-f753a4e5389d.png) ![Network Disconnected 48](https://user-images.githubusercontent.com/25880200/220018157-449d0c48-c57e-47bd-96d8-51066992d4ff.png) ![Network Disconnected 64](https://user-images.githubusercontent.com/25880200/220018160-78e7e839-1599-43f3-8be0-d86f3c1688b5.png) ![Network Disconnected 72](https://user-images.githubusercontent.com/25880200/220018162-cc3a6881-89e2-4ec7-84ff-689c6064757d.png) ![Network Disconnected 96](https://user-images.githubusercontent.com/25880200/220018163-55996ab8-941d-4a3b-9d6c-02282ac24c48.png)

**Comment by malekmasoud** [comment]
Here I am with updated Paintbrush icons with sizes ranging from 16x16 to 48x48!
![Paintbrush 16](https://user-images.githubusercontent.com/25880200/223012038-460c6f09-e60d-4d3b-9a57-ec3cb4b1ff6d.png) ![Paintbrush 22](https://user-images.githubusercontent.com/25880200/223012040-f5c3fe55-1c52-4cfb-a45e-38704517c6fa.png) ![Paintbrush 24](https://user-images.githubusercontent.com/25880200/223012041-9e4da6f2-d276-4080-8920-cedaefb8aaeb.png) ![Paintbrush 32](https://user-images.githubusercontent.com/25880200/223012043-d769f3d0-94ad-481e-8b5e-2e52cdda6b70.png) ![Paintbrush 48](https://user-images.githubusercontent.com/25880200/223012045-008f3a71-e5e9-434f-9e31-6331e1b49493.png)

**Reply by nestoris** [reply]
done! needed small correction (black/gray self-shadow border) now it's an app icon for `drawing`, `picopixel` and `kolourpaint`.

**Comment by malekmasoud** [comment]
Here is the actual icon from Microsoft Paint in Windows 98/2000/ME, with sizes ranging from 16x16 to 48x48.
![Paint 16](https://user-images.githubusercontent.com/25880200/232249332-5cbea374-03c2-43fa-a11e-02d773909ecc.png) ![Paint 22](https://user-images.githubusercontent.com/25880200/232249334-0470e9be-26a9-49f1-82df-955a4b21aa95.png) ![Paint 24](https://user-images.githubusercontent.com/25880200/232249335-8e6d880e-32d7-4d24-ab3e-afac00e7aa01.png) ![Paint 32](https://user-images.githubusercontent.com/25880200/232249336-35c31c15-0a21-443e-a2e6-9cb2e95c80be.png) ![Paint 48](https://user-images.githubusercontent.com/25880200/232249338-09bae43a-b200-441d-bb87-8994942d4301.png)

**Reply by nestoris** [reply]
for what do you want to use this icon?

**Reply by malekmasoud** [reply]
The MS Paint 98 (paintbrushes in a can) icon can be used as an app icon for `drawing` and `picopixel` whereas the 98-ified 3.1 Paintbrush (paintbrush and art palette) icon can be used as an app icon for `kolourpaint` and `style`, that way, both of these paint icons can be seen equally.
